lbz/misc.py

```diff
@@ -132,3 +133,29 @@ def deep_update(dict_to_update: dict, update_data: dict) -> None:
 
 def is_in_debug_mode() -> bool:
     return environ.get("LBZ_DEBUG_MODE", "").lower() == "true"
+
+
+def deprecated(message: str, version: str) -> Callable:
```

Comment:
I would allow using keyword arguments only:
```suggestion
def deprecated(*, message: str, version: str) -> Callable:
```